README.md

```diff
@@ -8,11 +8,9 @@ Teraslice processors for working with data stored in files on disk.
 | path | Directory where data will be saved. All intermediate directories must pre-exist.' | | Y |
 | file_prefix | A prefix to prepend to the file | 'export' | N |
 | fields | Array containing specific fields to include in the output | All fields | N |
-| delimiter | Character to use for separating fields in the output. | ',' | N |
```

Comment:
This should be preserved as an optional parameter just in case there's a use case that requires a different delimiter. `format` should configure this option by default but the user can override the setting if needed.